### ISSUE TITLE
Improve the Signature UI

### ISSUE THREAD

**Issue by Jzarecta** [spesmilo/electrum#1824]
I think this could recieve a little love, I use Kleopatra and their sign/verify experience is quite nice, they either have a wizard or a clipboard mode to sign/encrypt or verify singnatures. 

On the encrypt part it uses a pub (long) key, so it would be good if there is some type of combobox to get the key from the current addresses. This will make it easier to get specially if the action is executed from the Tools -> Menu as opposed to the context menu of the address which imports it ahead. 

I also think it would improve the look and feel if the messages are html/ize so it can add format to the message if desired.

**Comment by bauerj** [comment]
I don't think it would be a good idea to add HTML editing capabilities to Electrum.

However, changing the text input field to some kind of combo box would make sense.
